api/base/filters.py

```diff
@@ -60,7 +60,7 @@ def filter_queryset(self, request, queryset, view):
 class FilterMixin(object):
     """ View mixin with helper functions for filtering. """
 
-    QUERY_PATTERN = re.compile(r'^filter\[(?P<field>\w+)\](\[(?P<op>\w+)\])?$')
+    QUERY_PATTERN = re.compile(r'^filter\[(?P<fields>((?:,*\s*\w+)*))\](\[(?P<op>\w+)\])?$')
```

Comment:
I'm not a regex master, so maybe isolate this regex and run it through some unit tests. Or we can just make sure that the view/filter tests cover it, but if we want to be super-thorough about this regex, the fastest way is to make a non-osf test case series of tests that just gets thrown at this regex. I think that would be a wise plan.